src/common/flowTypes.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import { ITEM_TYPE_FOLDER, ITEM_TYPE_FILE, ITEM_TYPE_WEBLINK } from './constants';
+
+type InlineNoticeType = 'warning' | 'error' | 'success' | 'info';
+type ItemType = ITEM_TYPE_FOLDER | ITEM_TYPE_FILE | ITEM_TYPE_WEBLINK;
+
+type FileMini = {
+    id: string,
+    name: string,
+    type: 'file',
```

Comment:
If you change the constants as above, then this can be `typeof ITEM_TYPE_FILE` and the one below can be the folder counterpart